.abapgit.xml

```diff
@@ -1,7 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
guess this removes the byte order mark?

Reply:
Argh. I edited on GH. I can fix when I'm back home tomorrow